internal/monitors/utilization/utilization.go

```diff
@@ -0,0 +1,45 @@
+package utilization
+
+import (
+	"github.com/signalfx/signalfx-agent/internal/core/config"
+	"github.com/signalfx/signalfx-agent/internal/monitors"
+	"github.com/signalfx/signalfx-agent/internal/monitors/types"
+	log "github.com/sirupsen/logrus"
+)
+
+const monitorType = "signalfx-system-utilization"
+
+// MONITOR(signalfx-system-utilization):
+// This monitor reports utilization metrics for Windows
+// Hosts.  It is used to drive the Windows Smart Agent host navigator view and dashboard
+// content.
+//
+// Sample YAML configuration:
+//
+// ```yaml
+// monitors:
+//  - type: signalfx-system-utilization
+// ```
+
+var logger = log.WithFields(log.Fields{"monitorType": monitorType})
+
+// Config for this monitor
+type Config struct {
+	config.MonitorConfig `acceptsEndpoints:"false"`
```

Comment:
I would make this `singleInstance` (https://github.com/signalfx/signalfx-agent/blob/master/internal/monitors/metadata/hostmetadata/hostmetadata.go#L74) unless you know of a good use case for having multiple instances of this.